tidb/watch/no-down-peers.bash

```diff
@@ -0,0 +1,25 @@
+set -euo pipefail
+. "`cd $(dirname ${BASH_SOURCE[0]}) && pwd`/../../helper/helper.bash"
+
+env=`cat "${1}/env"`
+shift
+
+name=`must_env_val "${env}" 'tidb.cluster'`
+shift
+
+pd_leader_id=`must_pd_leader_id "${name}"`
+version=`env_val "${env}" 'tidb.version'`
+if [ -z "${version}" ]; then
+    version=`must_cluster_version "${name}"`
+fi
+
+# FIXME find a better way?
+# the default value of tikv down peer's heartbeat duration is 10m
+sleep 600   
```

Comment:
We could remove the long time sleeping, then sleep 1s in the `while` loop

Reply:
That does not work. In the default configuration, tikv can detect down peers after 10 minutes and report to PD. If we do not wait for 10 minutes, the number of down peers read from the PD is incorrect.

Reply:
For a testing flow ` ... : kill-or-jam-a-tikv : db.watch.no-down-kv : ...` things will be as you said.
How ever, `watch.no-down-kv` should not take response for "wait for kv down",

So, a better way:
`... : kill-or-jam-a-tikv : db.watch.has-down-kv : db.watch.no-down-kv : ...`
We could pull the status in 1s-interval, both in `db.watch.has-down-kv` and `db.watch.no-down-kv`